kyaml/setters2/add.go

```diff
@@ -48,17 +56,65 @@ func (a *Add) visitSequence(_ *yaml.RNode, _ string, _ *openapi.ResourceSchema)
 	return nil
 }
 
+// visitMapping implements visitor
+// visitMapping visits the fields in input MappingNode and adds setter/subst ref
+// if the path path spec matches with input FiledName
+func (a *Add) visitMapping(object *yaml.RNode, p string, _ *openapi.ResourceSchema) error {
+	return object.VisitFields(func(node *yaml.MapNode) error {
+		if node.Value.YNode().Kind != yaml.SequenceNode {
+			return nil
+		}
+
+		key, err := node.Key.String()
+		if err != nil {
+			return err
+		}
+
+		// derive the list values for the sequence node to write it to openAPI definitions
+		var values []string
+		for _, sc := range node.Value.Content() {
+			values = append(values, sc.Value)
+		}
+
+		// check if there are different values for field path
+		if len(a.ListValues) > 0 {
+			sort.Strings(values)
```

Comment:
I don't think we should do this. Sequences are ordered in yaml, so it is not clear that the same values in a different order should be considered equal.

Reply:
Done.